robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowBridgeContext.java.vm

```diff
@@ -0,0 +1,60 @@
+package org.robolectric.shadows;
+#if ($api == 21)
+import com.android.layoutlib.bridge.android.BridgeContext;
+import com.android.layoutlib.bridge.android.BridgePowerManager;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.robolectric.annotation.RealObject;
+import org.robolectric.util.ReflectionHelpers;
+
+import android.os.Handler;
+import android.os.PowerManager;
+import android.view.accessibility.AccessibilityManager;
+import android.view.textservice.TextServicesManager;
+
+import static org.robolectric.internal.Shadow.newInstanceOf;
+/**
+ * Shadow for {@link com.android.layoutlib.bridge.android.BridgeContext}
+ */
+@Implements(className = ShadowBridgeContext.CLASS_NAME)
+public class ShadowBridgeContext {
+  public static final String CLASS_NAME = "com.android.layoutlib.bridge.android.BridgeContext";
+
+  @RealObject private BridgeContext realContext;
+  
+  @Implementation
+  public Object getSystemService(String service) {
+    if ("layout_inflater".equals(service)) {
```

Comment:
Oh yeah, that makes sense then. Is there any reason why we couldn't reuse the service locator code from

https://github.com/robolectric/robolectric/blob/master/robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowContextImpl.java.vm ?